source/_static/badges/ent-cloud-only.rst

```diff
@@ -0,0 +1,22 @@
+:orphan:
```

Comment:
Thank you for creating a new badge for this Enterprise Cloud-only feature. Please note that the "only" badges are exception badges that display in yellow, and are typically used inline on a page where a general badge is included at the top of the page.

Please use the non-only badge code for a white plan/deployment badge instead.

Reply:
Latest commit should look like 
![image](https://github.com/mattermost/docs/assets/12176405/11622615-76ee-4ebb-a68e-8b5073f7bb30)